description: map insert doesn't support initializer list arguments in gcc 6
author: Michael Gilbert <mgilbert@debian.org>

--- a/services/image_annotation/annotator.cc
+++ b/services/image_annotation/annotator.cc
@@ -244,7 +244,7 @@ void Annotator::OnJpgImageDataReceived(
   std::unique_ptr<network::SimpleURLLoader> url_loader =
       MakeOcrRequestLoader(server_url_, source_id, image_bytes);
   const auto url_loader_it =
-      url_loaders_.insert({source_id, std::move(url_loader)}).first;
+      url_loaders_.insert(std::make_pair(source_id, std::move(url_loader))).first;
   url_loader_it->second->DownloadToString(
       url_loader_factory_.get(),
       base::BindOnce(&Annotator::OnServerResponseReceived,
